### ISSUE TITLE
[MINOR] Can we have a scroll bar on the side?

### ISSUE THREAD

**Issue by jason--liu** [org-roam/org-roam-ui#228]
When the org note is relatively long， it would be more convenient to have a scroll bar on the side in preview mode.

**Comment by meedstrom** [comment]
Agreed. My workaround is to select the text and drag, which is faster than my touchpad's scroll function on Chromium. I realize that some touchpads with inertial scrolling (Mac?) may make this a non-issue, but it's not universal.

A related issue: the scroll position is "sticky". If I scroll down in a big note and then click another note, it'll show that note scrolled way down too.

**Comment by ThomasFKJorna** [comment]
Hmm okay that is definitely not ideal! 
This might be an OS-specific issue, in both Firefox and Brave I see this (on MacOS)
<img width="512" alt="image" src="https://user-images.githubusercontent.com/21983833/161051091-3cdcc41d-ea67-47d0-8d61-3fc0b9d9e177.png">

What browser/OS are you on?

**Comment by jason--liu** [comment]
I'm on Ubuntu 20.04 LTS with Google chrome browser.
